src/presenters/teaser-presenter.js

```diff
@@ -110,6 +110,9 @@ const TeaserPresenter = class TeaserPresenter {
 			mods.push(hyphenatePascalCase(this.data.type));
 		}
 
+		if (this.data.type === 'LiveBlog') {
```

Comment:
Should we be checking the status === 'inprogress' here as well? i.e. a closed liveblog shouldn't go red...

Reply:
Fixed and I've also added a test https://github.com/Financial-Times/n-teaser/pull/146/commits/14ef5b39025762a38ebdef59558d68582348d934